st2api/setup.py

```diff
@@ -14,27 +14,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-try:
-    from setuptools import setup, find_packages
-except ImportError:
-    from ez_setup import use_setuptools
-    use_setuptools()
-    from setuptools import setup, find_packages
+import re
+import os.path
+from pip.req import parse_requirements
+from setuptools import setup, find_packages
+
+
+def fetch_requirements():
```

Comment:
`pip.req.parse_requirements` should work as long as a new version of pip is available on the system.

See https://github.com/tweepy/tweepy/issues/533 for an example.